packages/react-native-editor/package.json

```diff
@@ -1,123 +1,122 @@
-    "clean:runtime": "npm run clean:haste; npm run clean:react; npm run clean:metro; npm run clean:jest; npm run clean:watchman; npm run clean:babel-cache",
-    "clean:build-artifacts": "rm -rf ./ios/build && rm -rf ./ios/Pods",
-    "clean:aztec": "cd ../react-native-aztec && npm run clean",
-    "clean:haste": "rm -rf /tmp/haste-map-react-native-packager-*",
-    "clean:install": "npm run clean; npm install",
-    "clean:jest": "jest --clearCache --config ../../test/native/jest.config.js; rm -rf $TMPDIR/jest_*",
-    "clean:metro": "rm -rf $TMPDIR/metro-cache-*;",
-    "clean:react": "rm -rf $TMPDIR/react-*",
-    "clean:watchman": "command -v watchman >/dev/null 2>&1 && watchman watch-del-all; true",
-    "clean:babel-cache": "rm -rf ../../node_modules/.cache/babel-loader/*",
-    "clean:i18n-cache": "rm -rf ./i18n-cache/data/*.json && rm -f ./i18n-cache/index.native.js",
-    "version": "npm run bundle && git add -A bundle"
-  }
+	"name": "@wordpress/react-native-editor",
+	"version": "1.30.0",
+	"description": "Mobile WordPress gutenberg editor.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"react-native"
+	],
+	"private": true,
+	"config": {
+		"jsfiles": "./*.js src/*.js src/**/*.js src/**/**/*.js",
+		"scssfiles": "src/*.scss src/**/*.scss"
+	},
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/react-native-editor/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/react-native-editor"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=10",
+		"npm": ">=6.9"
+	},
+	"main": "src/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.9.2",
+		"@react-native-community/slider": "git+https://github.com/wordpress-mobile/react-native-slider.git#5ad284d92b8d886e366445bf215be741ed53ddc6",
+		"@wordpress/api-fetch": "file:../api-fetch",
+		"@wordpress/block-editor": "file:../block-editor",
+		"@wordpress/block-library": "file:../block-library",
+		"@wordpress/blocks": "file:../blocks",
+		"@wordpress/components": "file:../components",
+		"@wordpress/data": "file:../data",
+		"@wordpress/edit-post": "file:../edit-post",
+		"@wordpress/element": "file:../element",
+		"@wordpress/hooks": "file:../hooks",
+		"@wordpress/i18n": "file:../i18n",
+		"@wordpress/react-native-aztec": "file:../react-native-aztec",
+		"@wordpress/react-native-bridge": "file:../react-native-bridge",
+		"fast-average-color": "^4.3.0",
+		"jed": "^1.1.1",
+		"jsdom-jscore-rn": "git+https://github.com/iamcco/jsdom-jscore-rn.git#a562f3d57c27c13e5bfc8cf82d496e69a3ba2800",
+		"metro-react-native-babel-preset": "0.57.0",
+		"metro-react-native-babel-transformer": "0.56.0",
+		"node-fetch": "^2.6.0",
```

Comment:
I updated `node-fetch` to the latest version here and in `@wordpress/e2e-test-utils`.